src/plugins/ui_stateful/clipboard.ts

```diff
@@ -517,11 +555,31 @@ export class ClipboardPlugin extends UIPlugin {
     );
   }
 
-  private getHTMLContent(): string {
+  private async getHTMLContent(): Promise<string> {
     let innerHTML: string = "";
     const cells = this.copiedData?.cells;
     if (!cells) {
-      innerHTML = "\t";
+      if (this.copiedData?.figureId) {
+        const figureId = this.copiedData.figureId;
+        const figureSheetId = this.getters.getFigureSheetId(figureId)!;
+        const figure = this.getters.getFigure(figureSheetId, figureId)!;
+        if (figure.tag == "image") {
+          if (!this.fileStore) {
+            innerHTML = "\t";
+          }
```

Comment:
the if is kinda pointless without some kind of break/return/else. We still go through the code that follow and the next if re-set the innerHTML